plugin.py

```diff
@@ -0,0 +1,20 @@
+from lsp_utils import NpmClientHandler
+import os
+
+
+def plugin_loaded():
+    LspSassPlugin.setup()
+
+
+def plugin_unloaded():
+    LspSassPlugin.cleanup()
+
+
+class LspSassPlugin(NpmClientHandler):
+    package_name = __package__
+    server_directory = 'language-server'
+    server_binary_path = os.path.join(server_directory, 'node_modules', 'some-sass-language-server', 'bin', 'some-sass-language-server')
+
+    @classmethod
+    def required_node_version(cls) -> str:
+        return '>=20'
```

Comment:
It does not work with v18?
We are currently including 18 with lsp_utils so it would be quite inconvenient if it would work with that.

(that said, it's probably a good time to update to 20 in lsp_utils)

Reply:
I’m following what is defined in language server `package.json` engines field: https://github.com/wkillerud/some-sass/blob/main/packages/language-server/package.json#L14

I suppose it could work with Node 18, I know I’ve used it before I switched to Node 20.

Reply:
Then it's time to update node version in lsp_utils I think.

Reply:
Doing in https://github.com/sublimelsp/lsp_utils/pull/125.

We should release those at the same time (or lsp_utils first).

Reply:
I’ve update PR description to reflect this.